### PR TITLE
Add Bluesky embeds and related preference

### PR DESCRIPTION
**Pull request by shnrmn**

### CODE REVIEW

**Comment by shnrmn** [comment]
I went ahead and added Bluesky embeds. I'm Normy on the forums if that is necessary. I'm not too versed with Android dev so there may be issues but it does work and I tried to match the style of the other embed functions.

File: Awful.apk/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -403,6 +403,7 @@
     <!-- Post Embedding page -->
     <string name="post_embedding_description">Embed elements in posts instead of just showing a link. These options may cause scroll lag or graphical glitches</string>
     <string name="inline_tweets">Tweets</string>
+    <string name="inline_bluesky">Bluesky Posts</string>
```
Comment (Sereri) [inline Awful.apk/src/main/res/values/strings.xml:407]: Should've been Skeets smdh

**Review by Sereri — APPROVED** [review]
lgtm

Feel free to add yourself to `about_contributors_array` in strings.xml as well (if you want to)

**Comment by shnrmn** [comment]
> lgtm
> 
> Feel free to add yourself to `about_contributors_array` in strings.xml as well (if you want to)

Done! Only thing is the emdbeds don't use a dark theme and I don't see any way to impose it. I reached out to the Bluesky people so if you'd rather wait for a response we can hold off.

File: Awful.apk/src/main/res/values/strings.xml (hunk 1 above)
Comment (shnrmn) [reply Awful.apk/src/main/res/values/strings.xml:407]: 😭

**Pull request merged**